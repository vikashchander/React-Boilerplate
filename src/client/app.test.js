import React from 'react';
import { shallow } from "enzyme";
import  App  from "./App.js";

describe('Test App Entry point', function () {
  it('should have a header tag with Welcome to React Boilerplate App', function () {
    const wrapper =  shallow(<App />);
    expect(wrapper.find("h2").text()).toEqual("  Welcome to React Boilerplate App ");
    expect(wrapper).toMatchSnapshot();
  });
});