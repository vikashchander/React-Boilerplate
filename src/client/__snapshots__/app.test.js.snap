// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App Entry point should have a header tag with Welcome to React Boilerplate App 1`] = `ShallowWrapper {}`;
